content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md

```diff
@@ -174,7 +174,7 @@ PodSpec is a description of a pod.
 
 - **readinessGates** ([]PodReadinessGate)
 
-  If specified, all readiness gates will be evaluated for pod readiness. A pod is ready when all its containers are ready AND all conditions specified in the readiness gates have status equal to "True" More info: https://git.k8s.io/enhancements/keps/sig-network/0007-pod-ready%2B%2B.md
+  If specified, all readiness gates will be evaluated for pod readiness. A pod is ready when all its containers are ready AND all conditions specified in the readiness gates have status equal to "True" More info: https://git.k8s.io/enhancements/keps/sig-network/580-pod-readiness-gates/README.md
```

Comment:
This page is automatically generated. The place to fix it is actually [`pkg/apis/core/types.go`](https://github.com/kubernetes/kubernetes/blob/b0abe89ae259d5e891887414cb0e5f81c969c697/pkg/apis/core/types.go#L2835) in the main Kubernetes code.

@mkuger you could open an issue against [k/kubernetes](https://github.com/kubernetes/kubernetes) to ask about getting that fixed.